Source/WPEFramework/PostMortem.h

```diff
@@ -129,7 +152,7 @@ namespace PluginHost {
     public:
         PostMortemData(PostMortemData&&) = delete;
         PostMortemData(const PostMortemData&) = delete;
-        PostMortemData& operator= (const PostMortemData&) = delete;
+        PostMortemData& operator=(const PostMortemData&) = delete;
```

Comment:
I guess missing: 
`PostMortemData& operator=(PostMortemData&&) = delete;`

Reply:
As we discussed need to cross check

Reply:
Discussed it with Marcel and you're updates are correct. The idea is that indeed we do define/declare both the MoveConstructor *and* the MoveAssignment (as I expected) but we do not actively change *all* move contructors/move assignement operators, unless we touch the class. 
I misinterpreted theis change and "thought" you touched the class (whitespace changes) but that was probably due to some automatic formatting tool in your environment, So you did not "really" touch the calls and thus the MoveOperator was not mandatory to add...
So, no need to change this, consider the review remark as not made but if you touh a class alwasy add the Move Constructor and the Move Assignment, but I think you did that everywhere :-)

Reply:
done